index.html

```diff
@@ -12185,6 +12186,68 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-repeated">
+			<pdef>aria-repeated</pdef>
+			<div class="property-description">
+				<p>Indicates content is repeated for visual purposes.</p>
+				<p>The <code>aria-repeated</code> helps indicate that content is repeated, and that presenting the repeated content to assistive technology users could produce underesirable redundancy in the output. Unlike <pref>aria-hidden</pref>, <code>aria-repeated</code> content may be presented by screen readers to users when requested by the user.</p>
+				<p>Authors SHOULD only set <code>aria-repeated</code> to <code>true</code> on redundant instances of content because assistive technologies MAY skip
+			  over these instances during presentation to the user. Setting <code>aria-repeated</code> on a subtree indicates the entire subtree is redundant. If <code>aria-repeated</code> is true for a subtree, it cannot be reversed within that subtree by setting <code>aria-repeated</code> to false on a descendant.</p>
+ 				<p>Here are some examples of repeated content:</p>
+				<ul>
+					<li>Running headers and footer sections, repeated on each page</li>
+					<li>Repeated table header rows</li>
+					<li>Pull quote</li>
+				</ul>
+				<p>Because the repeated content may be important in some scenarios, e.g. during proofreading, assistive technologies MAY provide options so that screen readers users can choose whether or not to filter <code>aria-repeated</code> content out of the current view.</p>
```

Comment:
> repeated content **may** be important 

This instance of lowercase 'may' isn't normative, but we have recently decided to try not to use lowercase 'may', 'must' or 'should' in the prose just in case it confuses anyone. (There's usually a decent alternative word that doesn't mess up the meaning). So the above could be rewritten as either:
> repeated content might be important 

or:
> repeated content can be important